bash_completion.sh

```diff
@@ -128,11 +128,34 @@ _buildtest ()
 {
   local cur="${COMP_WORDS[COMP_CWORD]}"
   local prev="${COMP_WORDS[COMP_CWORD-1]}"
+  declare -i offset="0"
 
   COMPREPLY=()   # Array variable storing the possible completions.
 
-  next=${COMP_WORDS[1]}
+  declare -a buildtest_opts=("--color" "--config" "--debug" "--editor" "--help" "--helpcolor" "--logpath" "--loglevel" "--print-log" "--no-color" "--report" "--version" "--view-log" "-c" "-d" "-h" "-l" "-p" "-r" "-V")
 
+  commands=( "--color" "--config" "-c" "--report" "-r" "--loglevel" "-l" "--editor" )   # Array variable storing commands which require an input argument from the user.
```

Comment:
can you rename these variables it is confusing when using `commands` and then the `for command in ${COMP_WORDS[@]}"`

Reply:
@shahzebsiddiqui Done. I have added bash completion for `--editor` as well.

<img width="1302" alt="Screen Shot 2023-04-17 at 2 26 52 PM" src="https://user-images.githubusercontent.com/35829130/232615457-ee7423e3-7cbc-48c5-8c0b-8891bd4fc0ae.png">